src/binaryen-c.cpp

```diff
@@ -163,6 +163,32 @@ BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const
   return ret;
 }
 
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams) {
+  auto* wasm = (Module*)module;
+  auto* ret = new FunctionType;
```

Comment:
why not just do `FunctionType ret`?

(also maybe rename to `test` as it isn't returned)